src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -113,7 +113,7 @@ export class ExtensionsListView extends ViewletPanel {
 		this.list.layout(size);
 	}
 
-	async show(query: string): Promise<IPagedModel<IExtension>> {
+	async show(query: string, token?: CancellationToken): Promise<IPagedModel<IExtension>> {
```

Comment:
Why is token an optional parameter here?

Reply:
There are places where this method is being called and cancelation is not needed. So i added cancelation as optional behaviour.